assemblies/resources/bin/setenv

```diff
@@ -35,4 +35,9 @@
 # export KARAF_REDIRECT    # Enable/set the std/err redirection when using bin/start
 
 export EXTRA_JAVA_OPTS="-Dorg.eclipse.jetty.server.Request.maxFormContentSize=1500000 -Dfile.encoding=UTF-8"
+export JAVA_MAX_MEM=1G
+export JAVA_PERM_MEM=256M
 export KARAF_NOROOT=true
+
+
+. bin/check_ports "$KARAF_ETC"
```

Comment:
if you start the script from a different directory it now completely fails:

```
% ./build/opencast-dist-develop-8-SNAPSHOT/bin/start-opencast
…/opencast/build/opencast-dist-develop-8-SNAPSHOT/bin/setenv: line 43: bin/check_ports: No such file or directory
```

This include needs to be relative to `KARAF_HOME` or `KARAF_BASE` if I remember correctly… or relative to `start_opencast` I guess.

Reply:
fixed